developers/compute-to-data/compute-workflow.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Compute Workflow
+section: developers
+description: Understanding the Compute-to-Data (C2D) Workflow
+---
+
+🚀  Now that we've introduced the key actors and provided an overview of the process, it's time to delve into the nitty-gritty of the compute workflow. We'll dissect each step, examining the inner workings of Compute-to-Data (C2D). From data selection to secure computations, we'll leave no stone unturned in this exploration. 
+
+For visual clarity, here's an image of the workflow in action! 🖼️✨
+
+<figure><img src="../../.gitbook/assets/c2d/c2d_detailed_flow.png" alt=""><figcaption><p>Compute detailed flow diagram</p></figcaption></figure>
+
+Below, we'll outline each step in detail 📝
+
+## Starting a C2D Job
+1. The consumer initiates a compute-to-data job by selecting the desired data asset and algorithm. 
+2. The provider checks the orders on the blockchain.
+3. If the orders are valid, the provider can commence the compute flow.
```

Comment:
maybe worth mentioning the provider fees as well, this may be an option

```suggestion
3. If the orders for dataset, algorithm and compute to data fees are valid, the provider can commence the compute flow.
```